utils/contentful.ts

```diff
@@ -46,6 +46,9 @@ export type ContentfulSynth = {
   description: string;
   logo?: Entry<ContentfulLogo>;
   mintmanage: string;
+  externalApi?: string;
```

Comment:
this var is specific to defillama tvl, i would probably name it that way, unless you see a way we could support this as generalized api calls to wherever

Reply:
My original intention was to keep this variable universal and check the source of the api in the FE code if we'll have multiple API providers. Smth like `if (externalApi.includes(defi.llama) { doSmth() } else if (...) { doSmthElse() }`

Reply:
the problem is i dont see how we can parse the data without knowing the type of api its hitting

Reply:
One way is to deduct the type of the api based on the url structure